### ISSUE TITLE
DevTools doesn't work

### ISSUE THREAD

**Issue by supercrafter333** [pmmp/DevTools#53]
ERROR:

```[13:45:58] [Server thread/CRITICAL]: ErrorException: "Array and string offset access syntax with curly braces is deprecated" (EXCEPTION) in "plugins/PocketMine-DevTools.phar/src/DevTools/commands/GeneratePluginCommand" at line 109

[13:45:58] [Server thread/CRITICAL]: #0 pmsrc/vendor/pocketmine/classloader/src/BaseClassLoader(138): pocketmine\utils\Utils::errorExceptionHandler(integer 8192, string[69] Array and string offset access syntax with curly braces is deprecated, string[103] phar:///home/minecraft/plugins/PocketMine-DevTools.phar/src/DevTools/commands/Ge, integer 109, array[2])

[13:45:58] [Server thread/CRITICAL]: #1 pmsrc/vendor/pocketmine/classloader/src/BaseClassLoader(138): include()

[13:45:58] [Server thread/CRITICAL]: #2 pmsrc/vendor/pocketmine/classloader/src/BaseClassLoader(124): BaseClassLoader->loadClass(string[39] DevTools\commands\GeneratePluginCommand)

[13:45:58] [Server thread/CRITICAL]: #3 (): BaseClassLoader->{closure}(string[39] DevTools\commands\GeneratePluginCommand)

[13:45:58] [Server thread/CRITICAL]: #4 plugins/PocketMine-DevTools.phar/src/DevTools/DevTools(62): spl_autoload_call(string[39] DevTools\commands\GeneratePluginCommand)

[13:45:58] [Server thread/CRITICAL]: #5 pmsrc/src/pocketmine/plugin/PluginManager(182): DevTools\DevTools->onLoad()

[13:45:58] [Server thread/CRITICAL]: #6 pmsrc/src/pocketmine/plugin/PluginManager(346): pocketmine\plugin\PluginManager->loadPlugin(string[48] /home/minecraft/plugins/PocketMine-DevTools.phar, array[2])

[13:45:58] [Server thread/CRITICAL]: #7 pmsrc/src/pocketmine/Server(1530): pocketmine\plugin\PluginManager->loadPlugins(string[24] /home/minecraft/plugins/)

[13:45:58] [Server thread/CRITICAL]: #8 pmsrc/src/pocketmine/PocketMine(272): pocketmine\Server->__construct(object BaseClassLoader, object pocketmine\utils\MainLogger, string[16] /home/minecraft/, string[24] /home/minecraft/plugins/)

[13:45:58] [Server thread/CRITICAL]: #9 pmsrc/src/pocketmine/PocketMine(302): pocketmine\server()

[13:45:58] [Server thread/CRITICAL]: #10 pmsrc(11): require(string[68] phar:///home/minecraft/pocketmine.phar/src/pocketmine/PocketMine.php)

[13:45:58] [Server thread/CRITICAL]: Could not load plugin 'DevTools' ```

**Comment by dktapps** [comment]
Did it ever cross your mind to update the plugin before sending an issue? ...

**Comment by supercrafter333** [comment]
I've updated ja Plugin yet. But the same error was comming.

**Comment by dktapps** [comment]
You obviously haven't updated it because this was fixed 7 months ago and released in 1.14.1.
